style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url("./ic.jpeg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 5px;
+}
+
+.header {
+  display: inline;
+}
+
+.fa-refresh {
+  float: right;
+}
+
+#newtodo {
+  width: 94%;
+  height: 40px;
+  margin-bottom: 20px;
+}
+
+p {
+  display: inline;
+}
+
+.check {
+  text-decoration: line-through;
+  display: inline;
+}
+
+i {
```

Comment:
try giving class names or id names to the elements. Giving styles directly to elements may cause problems in scaling and maintaining the code